src/Servers/IIS/IIS/samples/NativeIISSample/Startup.cs

```diff
@@ -117,14 +118,22 @@ public void Configure(IApplicationBuilder app)
 
         public static void Main(string[] args)
         {
-            var host = new WebHostBuilder()
-                .UseKestrel()
-                .UseIIS()
-                .UseIISIntegration()
-                .UseStartup<Startup>()
-                .Build();
-
-            host.Run();
+            AppDomain.CurrentDomain.FirstChanceException += (sender, eventArgs) => {
+                Console.WriteLine("1");
+                NativeMethods.HttpSetStartupErrorPageContent("<pre>" + HtmlEncoder.Default.Encode(eventArgs.Exception.ToString()) + "</pre>");
```

Comment:
We need to decide the design of this page. Maybe reuse DeveloperExceptionPage code generation.

Reply:
Unfortunately, it has more dependencies then we might want.